quick_start.adoc

```diff
@@ -167,6 +171,21 @@ spec:
             - cn
           email:
             - mail
+
+----
+
+The `identityProviders` list contains the configurations for one or more identity providers. The example above contains a single identity provider (LDAP).
+An entry under `identityProviders` has the following fields:
+
+- `name` contains the unique name that is used to identify the identity provider.
+- `type` specifies the identity provider type and it is set to LDAP.
+- `mappingMethod` (add, claim or lookup) controls how mappings are established between this provider’s identities and User objects.
+- `ldap` contains the LDAP specific configurations:
+  * `url` contains the LDAP search parameters to use.
+  * `bindDN` contains an optional DN to bind with during the search phase.
```

Comment:
I don't think the bindDN is optional. Dex uses the bindDN and bindPW as credentials to connect to the LDAP directory and search for users.